src/en/proposals/revolution-rework.md

```diff
@@ -0,0 +1,166 @@
+
+It's contains additional comments, illustatrations, images and other MAYBE useful information.
+
+**RUS VERSION --->** <https://docs.google.com/document/d/11Lpip1kL9xyVAVoQn-PVuqBHv-stovz8ZaweRDGwxtE/edit?usp=sharing>
+
+## 1. GENERAL MECHANICS
+### 1.1 Rev Uplink
+- Headrevs have another distinctive feature, the rev **uplink implant**. It contains a variety of often decrepit, rusty and makeshift equipment.
+- For **one recruited** it gives **1 Rebel Telecrystal** (RTC) to headrev’s uplink.
+> A full list of gear is being worked on.
+Approximately:
+Soviet-made pistols, Mosinky, PPSh, Kalashnikovs, RGDs, various body armor, clothing, riot posters, homemade war makings, ammunition and other paraphernalia
+
+**FOR WHAT:** These things will make the headrev stand out from the crowd or help him in some minor actions for later climax (like raiding an security patrol).
+
+### 1.2 Holographic Armband
+- In exchange for the icons, the revolutionary will be given the **ability "make intentions known"** which will create a red or blue-red **armband** on his shoulder for the headrevs.
+- The **ability is one-time** and there is no way to remove this bandage once activated.
+- To know the player's affiliation before activation requires examining the character - it will answer it in examine window (being a revolutionary).
+- Once alerted _(1.6)_, all armbands automatically appear on the shoulders of the rebels.
+
+**FOR WHAT:** To get rid of stealthy revolutions, add "rebel entourage" and replace trivial meta icons.
+
+### 1.3 Mindshield
+Has no effect on revolutionaries, but prevents further recruitment _(1.8)_.
+
+### 1.4 Goldstein Books 
+- There is a **free** item in headrev uplink _(1.1)_ - a **“brainwash book”.**
+- Initially there is no books in the uplink _(1.1)_, but for every **[headrev count * 2]** revolutionary, the copy will be replenished by one piece. **[Maximum books = 6]**.
+- When read **(by pressing Z)**, the character becomes a revolutionary. If a loyalist or rebel does it, nothing happens.
+>Hereinafter, a loyalist is a character with a mindshield implant. In the points system, a loyalist and a head are different people.
+
+**FOR WHAT:** Diversify the method of recruitment, allow simple revolutionaries to recruit, and make a reference to 1984.
+
+### 1.5 Rebel Flag
+- There is another free item in headrev uplink _(1.1)_ - rebel flag, which **can be placed** in different departments.
+- Used exclusively for the points system _(3.2)_.
+- If the department contains **[number of loyalists / 5]**, it is **automatically removed** after a minute.
+- Normal crew can remove it manually activating the process for 1 minute (can't move). Can't be moved.
+- **Can't be staged after** the evacuation shuttle has been dispatched.
+
+**FOR WHAT:** to get rid of stealthy revolutions, diversify the strategic part of the mode, and supplement the points system.
```

Comment:
I think getting rid of stealthy revolutions is the wrong idea. Revs are intended to be a primarily social deduction (Stealth) gamemode where the crew try figure out the imposters among them (SUS), not just another nukie gamemode. I do agree that having alternative ways for revs to take over the station is good.